presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -680,19 +680,22 @@ else if (existingPrivilege.isContainedIn(requestedPrivilege)) {
                     .run("grantTablePrivileges", stats.getGrantTablePrivileges().wrap(() -> {
                         try (HiveMetastoreClient metastoreClient = clientProvider.createMetastoreClient()) {
                             PrincipalType principalType;
+                            String principalName;
 
-                            if (metastoreClient.getRoleNames().contains(grantee)) {
+                            if (metastoreClient.getRoleNames().contains(grantee.toLowerCase())) {
```

Comment:
It would be nice if you abstracted this if() to a method that was shared between the grant and revoke functions, but I won't insist on that.